packages/image_picker/image_picker_for_web/lib/src/pkg_web_tweaks.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:web/web.dart' as web;
+
+/// Adds a `toList` method to [web.FileList] objects.
+extension WebFileListToDartList on web.FileList {
+  /// Converts a [web.FileList] into a [List] of [web.File].
+  ///
+  /// This method makes a copy.
+  List<web.File> get toList =>
```

Comment:
meh...I'd opt for a function here. But maybe that's just my preference.

Reply:
I hope this eventually gets provided by package:web, as some form of iterable thing, so I don't need any of my code :P